docs/extend/modular-resources/create/_index.md

```diff
@@ -493,15 +473,22 @@ For example:
 #!/bin/sh
 cd `dirname $0`
 
+# Create a virtual environment to run our code
+VENV_NAME="venv"
```

Comment:
isn't `venv` supplied with all pythons 3.3 and above? https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv, since we only support >3.8, they should come with python if it is installed

Reply:
You're right, thank you! I withdraw that comment!